pf/go.mod

```diff
@@ -11,9 +11,9 @@ require (
 	github.com/blang/semver v3.5.1+incompatible
 	github.com/golang/protobuf v1.5.4
 	github.com/hashicorp/go-multierror v1.1.1
-	github.com/hashicorp/terraform-plugin-framework v1.6.0
+	github.com/hashicorp/terraform-plugin-framework v1.7.0
```

Comment:
This is the minimum necessary upgrade, but we're quite behind. We should catch up with the latest version.